Solar Auxilia.cat

```diff
@@ -2929,7 +2929,7 @@
         <categoryLink targetId="594d-fa82-13cb-a345" id="640b-a42f-1aa2-dc90" primary="false" name="Infantry Model Type"/>
       </categoryLinks>
       <selectionEntries>
-        <selectionEntry type="model" import="true" name="Rapier Crew" hidden="false" id="b61d-cda1-3891-2dae" collective="false" flatten="false" sortIndex="1">
+        <selectionEntry type="unit" import="true" name="Rapier Crew" hidden="false" id="b61d-cda1-3891-2dae" collective="false" flatten="false" sortIndex="1" subType="unit-group">
```

Comment:
I'd almost rather this just be "upgrade" but maybe unit group is right, I'll have to look at it

Reply:
It's a weird unit on how the book lays it out, I tried three ways and nothing seemed to be great.